_opensearch/snapshots/searchable_snapshot.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: default
+title: Searchable snapshots
+parent: Snapshots
+nav_order: 40
+has_children: false
+---
+
+# Searchable snapshots
+
+Searchable snapshots is an experimental feature with OpenSearch 2.4. Therefore, we do not recommend the use of this feature in a production environment. For updates on progress or if you want leave feedback that could help improve the feature, see the [searchable snapshot GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/2919). 
+{: .warning }
+
+A searchable snapshot index is an index where data is read from a [snapshot repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/#register-repository) on-demand at search time, rather than downloading all index data to cluster storage at restore time. Because the index data remains in the snapshot format in the repository, searchable snapshot indexes are inherently read-only. Any attempt to write to a searchable snapshot index will result in an error.
+
+As an experimental feature, searchable snapshots will be behind a feature flag.
```

Comment:
I'm open to be convinced, but I don't think we need this note here considering the warning at the top.

Reply:
yeah, after shorting the note I agree, we can just remove it